sources/lib/DependencyInjection/PommExtension.php

```diff
@@ -60,11 +62,42 @@ public function configure(array $configs, ContainerBuilder $container)
         $configuration = new Configuration();
         $config = $processor->processConfiguration($configuration, $configs);
 
-        $container->setParameter('pomm.configuration', $config['configuration']);
+        $definition = $container->getDefinition('pomm');
+        foreach ($config['configuration'] as $name => $pommConfig) {
+            if (isset($pommConfig['service']) && isset($pommConfig['class:session_builder'])) {
+                throw new \RuntimeException(sprintf('Please configure either "service" or "class:session_builder" parameter on %s', $name));
```

Comment:
validating this is the job of the configuration class. See https://github.com/symfony/symfony/blob/v3.2.1/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php#L502 for an example